src/Zoomba/GUILibrary.py

```diff
@@ -32,109 +33,122 @@ def element_value_should_not_be_equal(self, locator, expected_value):
         zoomba.should_not_be_equal(value, expected_value)
 
     @keyword("Wait For And Click Element")
-    def wait_for_and_click_element(self, locator):
+    def wait_for_and_click_element(self, locator, timeout=None):
```

Comment:
Is there a way to default the timeout value to it's default instead of None? I think setting this to None means that everyone is going to have to edit all uses of these keywords to include some timeout value.

Reply:
Even our library sets the default to timeout=5.0

Reply:
Let me double check but I am pretty sure 'timeout=None' is what the keyword defaults to anyway when there is no override. In which case it uses the default set by 'Set Selenium Timeout'.

Reply:
Thanks again for the explanation and demo today